src/screens/APIUtils/APIUtils.res

```diff
@@ -176,9 +176,7 @@ let getURL = (
     | #ACCEPT_INVITE => `${userUrl}/user/invite/accept`
     | #USER_DELETE => `${userUrl}/user/delete`
     | #UPDATE_ROLE => `${userUrl}/user/${(userType :> string)->String.toLowerCase}`
-    | #SIGNIN
```

Comment:
Can we verify that we dont need these anymore in any case

Reply:
Yes. We don't need